pkg/cache/informer_backed_test.go

```diff
@@ -32,6 +33,11 @@ var (
 			},
 		}
 	})
+	// dummySecret1 represents a dummy Secret resource. the base64 data is 'hello world'
+	dummySecret1 = secrettestutil.DummySecretResource("namespace-1", "name-1", func(secret *corev1.Secret) {
+		secret.Data["tls.crt"] = []byte("aGVsbG8gd29ybGQK")
+		secret.Data["tls.key"] = []byte("aGVsbG8gd29ybGQK")
```

Comment:
Let's use distinct values for cert and key so we are sure correct fields are used in correct order.